### PR TITLE
WL-3444: Get original site updaters from tool session

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -93,7 +93,11 @@
 			<groupId>org.sakaiproject.entitybroker</groupId>
 			<artifactId>entitybroker-utils</artifactId>
    		</dependency>
-        <dependency>
+		<dependency>
+			<groupId>org.sakaiproject.portal</groupId>
+			<artifactId>sakai-portal-impl</artifactId>
+			</dependency>
```
Comment (ghost) [inline pom.xml]: Indentation

**Comment by nicholaswilson100** [comment]
Removed dependency on portal impl. Optimise imports.

**Comment by nicholaswilson100** [comment]
```
                       Thanks
```

(get it?)
(parting joke)

**Comment by ghost** [comment]
HILARIOUS

**Pull request merged**